pkg/util/oc/pod_commands.go

```diff
@@ -61,8 +62,12 @@ func WaitPodReady(t test.TestHelper, podLocator PodLocatorFunc) {
 	retry.UntilSuccess(t, func(t test.TestHelper) {
 		pod = podLocator(t)
 	})
-
-	shell.Executef(t, "kubectl -n %s wait --for condition=Ready pod %s --timeout 30s", pod.Namespace, pod.Name)
+	condition := shell.Executef(t, "kubectl -n %s wait --for condition=Ready pod %s --timeout 30s", pod.Namespace, pod.Name)
+	if strings.Contains(condition, "condition met") {
+		t.Logf("Pod %s in namespace %s is ready!", pod.Name, pod.Namespace)
+	} else {
+		t.Fatalf("Error: %s in namespace %s is not ready: %s", pod.Name, pod.Namespace, condition)
```

Comment:
This line is never reached, because `shell.Executef` fails the test when the command exits with a non-zero exit code (which is what happens when the wait command times out).

Reply:
ok, let's go with `|| true` or `|| echo "timed out"` for now.